wear/src/test/kotlin/io/homeassistant/companion/android/data/WearDnsTest.kt

```diff
@@ -0,0 +1,43 @@
+package io.homeassistant.companion.android.data
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import io.homeassistant.companion.android.data.ShadowDnsSystem.Companion.shadowOf
+import java.net.InetAddress
+import java.net.UnknownHostException
+import okhttp3.Dns
+import org.junit.Test
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.ConscryptMode
+
+@RunWith(RobolectricTestRunner::class)
+@Config(shadows = [ShadowDnsSystem::class])
+@ConscryptMode(ConscryptMode.Mode.ON)
+class WearDnsTest {
+
+    private val context = ApplicationProvider.getApplicationContext<Context>()
+    private val dns = WearDns(FakeMessageClient(context), FakeCapabilityClient(context))
+
+    val homeAssistantLocal = InetAddress.getByAddress("homeassistant.local", byteArrayOf(192.toByte(), 168.toByte(), 0, 23))
+
+    @Test
+    fun `defaults to system dns`() {
+        val shadowDns = shadowOf(Dns.SYSTEM)
```

Comment:
@TimoPtr this is the approach for unit testing, does it seem ok to you?

Reply:
Yes it's fine 👍🏻 You could also use `mockk` if you want if it makes things a bit easier to setup.

Reply:
Does mockk replace Robolectric shadows?